django/db/backends/postgresql_psycopg2/features.py

```diff
@@ -27,3 +28,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     closed_cursor_error_class = InterfaceError
     has_case_insensitive_like = False
     requires_sqlparse_for_splitting = False
+
+    @cached_property
+    def allows_group_by_selected_pks(self):
+        return self.connection.pg_version >= 90100
```

Comment:
This might not be required if we drop support for 9.0 in 1.9. @shaib pointed out that it will EOL in September.